### DISCUSSION TITLE
Turning-off notification

### DISCUSSION THREAD

**Discussion by technicznyblog** [EpicenterHQ/epicenter#365] [General]
Would it be possible to disable notifications? Unfortunately, in Linux, specifically Ubuntu, these notifications pop up at the top and can be very annoying, especially when something goes wrong with pasting text from the clipboard. It would be great to have an option to completely turn off any notifications. Unfortunately, I couldn't find any option to disable this on system level easily.
![image](https://github.com/user-attachments/assets/a32ad38e-f3ee-4ecb-a44b-b50f17c8ba87)

**Comment by rkhrkh** [comment]
On macOS, there's a similar issue, but I simply resize the Whispering window to about the size of a thumb. This way, I can still see the icon but no longer the notifications.
